lib/birdie/account/account.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Birdie.Account do
```

Comment:
I think we can put the functions here into the `user` module. What do you think? @mattong